package/native-package/yarn.lock

```diff
@@ -3670,27 +3569,28 @@ react-native-url-polyfill@^1.3.0:
   dependencies:
     whatwg-url-without-unicode "8.0.0-3"
 
-react-native@>=0.73.6:
-  version "0.74.2"
-  resolved "https://registry.yarnpkg.com/react-native/-/react-native-0.74.2.tgz#172e64e4e79861e2b3da99b86999e4a4c55b8321"
-  integrity sha512-EBMBjPPL4/GjHMP4NqsZabT3gI5WU9cSmduABGAGrd8uIcmTZ5F2Ng9k6gFmRm7n8e8CULxDNu98ZpQfBjl7Bw==
+react-native@^0.73.6:
+  version "0.73.8"
```

Comment:
this has been downgraded. is this what we expect?

Reply:
Yes it's good for now, until we test with 0.74 on native cli